### DISCUSSION TITLE
Added Circles and Polygons to Maps

### DISCUSSION THREAD

**Discussion by edburley** [CommunityToolkit/Maui#1647] [New Feature Discussions]
Circle and Polygon drawing was missing from CommunityToolkit.Maui.Maps/Handler/Map/MapHandler.Windows.cs.

This patch adds both Circles and Polygons.
The erase method for pins had to be modified to not clear things that aren't pins.

Usage: 
                MyMap.MapElements.Clear();
                Circle circle = new Circle
                {
                    Center = center,
                    Radius = Distance.FromMiles(circleRadius),
                    StrokeColor = Color.FromArgb("#88FF0000"),
                    StrokeWidth = 3,
                    FillColor = Color.FromArgb("#88FFC0CB")
                };
                MyMap.MapElements.Add(circle);

[0001-MapHandler-Windows-cs.patch](https://github.com/CommunityToolkit/Maui/files/13955137/0001-MapHandler-Windows-cs.patch)

**Comment by CliffAgius** [comment]
Closing as this is now implemented you can read the docs here:

https://learn.microsoft.com/en-us/dotnet/maui/user-interface/controls/map?view=net-maui-8.0#polygons-polylines-and-circles
